RecoTauTag/HLTProducers/src/HLTPFDiJetCorrCheckerWithDiTau.cc

```diff
@@ -134,3 +143,5 @@ void HLTPFDiJetCorrCheckerWithDiTau::fillDescriptions(edm::ConfigurationDescript
 //
 #include "FWCore/Framework/interface/MakerMacros.h"
 DEFINE_FWK_MODULE(HLTPFDiJetCorrCheckerWithDiTau);
+using HLTDiPFJetPlusTausFilter = HLTPFDiJetCorrCheckerWithDiTau;
+DEFINE_FWK_MODULE(HLTDiPFJetPlusTausFilter);
```

Comment:
```suggestion
using HLTDiPFJetPlusTausPFJetProducer = HLTPFDiJetCorrCheckerWithDiTau;
DEFINE_FWK_MODULE(DiPFJetPlusTausPFJetProducer);
```
I have to change my earlier suggestion (sorry..). This is not a good name, as this is an EDProducer (not an EDFilter). Feel free to suggest a better name.

Reply:
In general, I would use `*PFJetProducer` (as opposed to `*DiPFJetProducer`), as the output is a list of jets (not a list of jet pairs). Words like "compute", "decorrelate" or "clean" in the plugin name are either too obvious or vague, imho.

`HLTDiPFJetPlusTausPFJetProducer` is shorter (pro). It is more vague (con), but this could also help in the future (pro) if the producer is extended (with extra options, extra cuts, etc). Example: in the future someone might want to use this plugin turning off the Mjj cut (can be done with `mjjMin = -1`), and adding the option to cut on something else, "X(j,j)"; in that scenario, having "Mjj" in the plugin name is not great.

The idea is that this outputs the jets that are part of "good DiJet + Tau(s) candidates". Maybe something like `HLTDiPFJetPlusTausTopologyPFJetProducer` or `HLTDiPFJetPlusTausCandidatePFJetProducer` might be clearer (I tend to favour the latter).

Reply:
I see your point. I agree and think that ```HLTDiPFJetPlusTausCandidatePFJetProducer``` is more future proof. I'll make the change and commit shortly

Reply:
Sounds good. You might as well squash again in 1 commit at the next (force) push.